src/core/logger/formatters.ts

```diff
@@ -7,6 +7,13 @@ import { MESSAGE } from 'triple-beam';
 import { inspect } from 'util';
 import { format } from 'winston';
 
+interface Trace {
+  getFunctionName(): string;
+  getFileName(): string;
+  getLineNumber(): number;
+  getColumnNumber(): number;
+}
```

Comment:
Pull types from library
```
yarn add -D @types/stack-trace
```

Reply:
Got it! Fixed locally, will push up soon.